low-level/json_msgs/messages/base_msg.py

```diff
@@ -64,7 +64,8 @@ def prepare_message(self, jsonMsg, message_type):
             if payload.get(f"message>{message_type}>info>cluster_id") is None:
                 jsonMsg["message"][message_type]["info"]["cluster_id"] = Conf.get(
                     GLOBAL_CONF, CLUSTER_ID_KEY, DEFAULT_CLUSTER)
-
+            if payload.get(f"message>{message_type}>info>fru") is None:

```

Comment:
confirm this function not over riding fru value what every sensor would set?

Reply:
It will not override the fru value if sensor is parsing the value.
Though, if nothing is parsed to json, then by default it will set fru:false for the sensors/actuators

Reply:
Do we really need to add fru field for all resources?
How it is useful for ex. node:sw:os:service ?

Reply:
@sandeepsgt  as discussed, to keep the alert schema same for the sensors, we will keep fru flag for all the sensors with default val as false